stable/nginx-ingress/values.yaml

```diff
@@ -10,7 +10,7 @@ controller:
     #  asia.gcr.io
     registry: us.gcr.io
     repository: k8s-artifacts-prod/ingress-nginx/controller
-    tag: "v0.34.0"
+    tag: "v0.34.1"
     # digest: sha256:0e072dddd1f7f8fc8909a2ca6f65e76c5f0d2fcfb8be47935ae3457e8bbceb20
```

Comment:
Though commented out, would you please update the digest to latest as well?
`sha256:0e072dddd1f7f8fc8909a2ca6f65e76c5f0d2fcfb8be47935ae3457e8bbceb20`

Reply:
The digest already has this value. v0.34.1 was likely already released at the time the helm chart was previously updated.

Reply:
Indeed, thanks! 😎